File: EgoCapture/Services/Upload/EmbeddedAWSCredentials.swift
import Foundation

/// Public-facing AWS credentials used by the upload pipeline.
///
/// The access key / secret are kept in a sibling file `EmbeddedAWSSecrets.swift`
/// which is **gitignored**. If you just cloned this repo, copy
/// `EmbeddedAWSSecrets.swift.example` to `EmbeddedAWSSecrets.swift` and fill in
/// the real IAM credentials before building.
enum EmbeddedAWSCredentials {
    static let bucket = "kaivideo"
    static let region = "us-east-1"

    static let accessKeyId = EmbeddedAWSSecrets.accessKeyId
    static let secretAccessKey = EmbeddedAWSSecrets.secretAccessKey
}

File: android/app/src/main/java/com/kgeneye/eye/upload/EmbeddedAwsCredentials.kt
package com.kgeneye.eye.upload

/**
 * Public-facing AWS credentials used by the upload pipeline.
 *
 * The access key / secret live in a sibling file `EmbeddedAwsSecrets.kt`
 * which is **gitignored**. If you just cloned this repo, copy
 * `EmbeddedAwsSecrets.kt.example` to `EmbeddedAwsSecrets.kt` and fill in
 * the real IAM credentials before building.
 */
object EmbeddedAwsCredentials {
    const val BUCKET = "kaivideo"
    const val REGION = "us-east-1"

    val accessKeyId: String get() = EmbeddedAwsSecrets.accessKeyId
    val secretAccessKey: String get() = EmbeddedAwsSecrets.secretAccessKey
}
